mmv1/products/redis/Cluster.yaml

```diff
@@ -188,6 +188,20 @@ examples:
       'deletion_protection_enabled': 'false'
     oics_vars_overrides:
       'deletion_protection_enabled': 'false'
+  - name: "redis_cluster_cmek"
+    primary_resource_id: "cluster-cmek"
+    vars:
+      cluster_name: "cmek-cluster"
+      policy_name: "mypolicy"
+      subnet_name: "mysubnet"
+      network_name: "mynetwork"
```

Comment:
These should all contain a `-` or `_` so that they get sweepable prefixes, as described in https://googlecloudplatform.github.io/magic-modules/test/test/#add-a-create-test. Otherwise, resources that leak over time won't be possible to clean up. It would be great to also update the other examples in this file (though that could be a separate PR).

Reply:
done